docs/zeta_integration.adoc

```diff
@@ -0,0 +1,85 @@
+= Zeta Integration Design
+
+== Architecture Overview
+
+image::images/zeta_integration_high_level.jpg["highlevel Architecture", width=800, link="zeta_integration_high_level.jpg"]
+
+1. Control/API Network. Alcor server configures the group entries and the buckets in the group entries for each computing node. 
+Each group entry corresponds to a Zeta Gateway Cluster (ZGC). Each bucket in the group entry corresponds to a gateway in ZGC.
+
+2. Tenant Network. The gateway node in ZGC is responsible for delivering the rules to ACA on computing nodes when two VMs communicate for the first time. 
+ACA as the openflow controller will install rules on ovs.
+
+3. Gateway/Direct path. The path between the gateway node and the computing node is called the gateway path whereas VM to VM direct communication is called direct path.
+
+== Bucketing System Design on Compute Node
+
+The information of the ZGC cluster will be when to computing node when needed from Alcor controller. When a VM boots up, 
+if the compute node already have a VM with the same VPC, there is no need to configure flow table and group table.
```

Comment:
We will look each VPC (which has its own VNI) will use the same ZGC. Therefore, I updated the wording above. Internally, ACA uses an internal VLAN ID to seperate tenants traffic. Our openflow rule will match the internal VLAN ID to determine which group table to go. We need to figure out when to clean up the group table. Can not simply delete it when VM port is deleted because the group table can be shared.